### ISSUE TITLE
Employee Activation 

### ISSUE THREAD

**Issue by prupapara** [AAndharia/iManager#156]
Employee Activation  in login page.

**Comment by prupapara** [comment]
Done Please verify.
@AAndharia  I have Implemented employee activation process in login and authsvc.js and etc. files.
Please verify once. commit no is e0fcc18

**Comment by jmervana** [comment]
1. in change password screen, see video : http://screencast.com/t/wvZXi4yoJ

2. in 2nd page, just try to change url. it will redirect you to dashboard. i should stay on same page.

3. if I go back from dashboard, it will redirect to change password screen which was already done. it should redirect me to 2nd step of employee activation.

4. see screenshot - http://screencast.com/t/fNweCywRAE, I think this issue is because of not define max length in html. give max length to address section.

5. I have checked "are you married" checkbox and select date, after that I have unchecked checkbox and click on next save marrige aniversary date and same as **"passport number"** 
see screenshots : - http://screencast.com/t/nm3owdCmN & http://screencast.com/t/AQXLUOvT

6. future date should be disabled for DOB

7. Add checkbox in 3rd step to ask user have you any exp?
    if checkbox checked  - minimum one employeement record is mandatory
    if not checked - user can finish step.

8. Add former employeement modal popup - joining date and reliving date , restrict user to select           future date.

9. see screenshot : http://screencast.com/t/IAs8FSlFF

10. 2nd & 3rd point above is also for step 3.
      after completed step 1 & step 2 again , step 3 is as it is either all step stay as it is or all step reset

11. after complete step 3, I redirected to dashboard which is good but when I try to change url from      
      dashboard to **"#/add-emp-personal-info"** it will again redirect me to first step of employee     
      activation. I can access this URL even if I completed this steps.

12. dashboard is empty for me after all step completed. just logout option is available for me
      see screenshot: http://screencast.com/t/FG50TLtsyTqF

13. after logout when I came to dashboard - and try to access activation URL , I can access this       URL.

**Comment by prupapara** [comment]
1 is Done Please verify.

**Comment by prupapara** [comment]
2 Already fixed.

**Comment by prupapara** [comment]
3. already done.

**Comment by prupapara** [comment]
4. Done Please verify.

**Comment by prupapara** [comment]
4 Done Please verify.

**Comment by prupapara** [comment]
5 Done Please verify.

**Comment by prupapara** [comment]
6 Done

**Comment by prupapara** [comment]
8 Done

**Comment by prupapara** [comment]
Some Task are already Done.
and other reaming task are implemented. 
Done Please verify.

**Comment by jmervana** [comment]
@prupapara 
1. When I am on 2nd step and refresh page , I will redirect to dashboard.

2. After redirecting to dashboard, I logged out and again logged in. now I should be redirect to 2nd step but I redirected to dashboard.

3. if I fill Address1 for Present Address and then I will check same as Permanant , it should save only one address and present and permanant address id should be same for an employee but it saves two different address and save different address Id for an employee

4. future date should be disable for aniversary date

5. see screenshot : http://screencast.com/t/wZEn1Xaw

6. When I try to add employment record in modal popup , I got an error
    see video : http://screencast.com/t/4NU8yZA6K6
